app/services/notifier/facebook_messenger_notifier.rb

```diff
@@ -1,3 +1,5 @@
+# Remove the next line this line when you (yes you) start working on this file
+# rubocop:disable all
```

Comment:
Why is the disable all necessary?

Reply:
File has unused variables in multiple places at least, and I think possibly some other things.

Reply:
Fair enough.